docs/book/src/reference/versions.md

```diff
@@ -63,17 +63,17 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                  |  CAPI v0.3 (v1alpha3) |  CAPI v0.4 (v1alpha4)  |  CAPI v1.0 (v1beta1) | CAPI v1.1+v1.2 (v1beta1) |
-| ---------------- | --------------------- | ---------------------- | -------------------- | ------------------------ |
-| Kubernetes v1.16 | ✓                     |                        |                      |                          |
-| Kubernetes v1.17 | ✓                     |                        |                      |                          |
-| Kubernetes v1.18 | ✓                     | ✓ (only workload)      | ✓ (only workload)    | ✓ (only workload)        |
-| Kubernetes v1.19 | ✓                     | ✓                      | ✓                    | ✓                        |
-| Kubernetes v1.20 | ✓                     | ✓                      | ✓                    | ✓                        |
-| Kubernetes v1.21 | ✓                     | ✓                      | ✓                    | ✓                        |
-| Kubernetes v1.22 | ✓ (only workload)     | ✓                      | ✓                    | ✓                        |
-| Kubernetes v1.23* |                      | ✓                      | ✓                    | ✓                        |
-| Kubernetes v1.24 |                       |                        |                      | ✓                        |
+|                   | CAPI v0.3 (v1alpha3) | CAPI v0.4 (v1alpha4) | CAPI v1.0 (v1beta1) | CAPI v1.1+v1.2 (v1beta1) |
```

Comment:
I know I said this before, but it's worth repeating. The GitHub UI diff is so so so bad

Reply:
Yeah - IDE keeps telling me to reformat these tables - but it's impossible to see what the diff is here ( I think it's probably one tab somewhere)

Reply:
I diffed it with Intellij, it's fine :)